src/core/Chip/styles.css

```diff
@@ -0,0 +1,85 @@
+.container {
+    align-items: center;
+    background-color: #eceff1;
+    border-radius: 16px;
+    cursor: pointer;
+    display: inline-flex;
+    font-size: 14px;
+    height: 32px;
+    margin: 4px;
+    user-select: none;
+}
+
+.container:hover {
+    background-color: #e0f2f1;
+}
+
+.disabled {
+    cursor: not-allowed;
+    opacity: 0.3;
+}
+
+.disabled .remove-icon {
+    pointer-events: none;
+}
+
+.dragging {
+    box-shadow: 0 3px 1px -2px rgba(0, 0, 0, 0.2),
+        0 2px 2px 0 rgba(0, 0, 0, 0.14), 0 1px 5px 0 rgba(0, 0, 0, 0.12);
+}
+
+.icon {
+    color: #494949;
+    font-size: 20px;
+    margin-left: 5px;
```

Comment:
Icon
- `margin-left` is `8px`
- Icon area is 24px, but star is 16px in size with 4px padding

Reply:
Checked this with Joe, my styles are OK, the specs are outdated.

Reply:
![2018-10-18-170226_326x267_scrot](https://user-images.githubusercontent.com/185449/47164105-a1511200-d2f7-11e8-8438-5a7fd2ef918f.png)

The ones coded are the right ones atm.